lib/discordrb/data/server.rb

```diff
@@ -620,10 +620,11 @@ def kick(user, reason = nil)
     end
 
     # Forcibly moves a user into a different voice channel. Only works if the bot has the permission needed.
+    # Can't move user if he's not already connected to a server voice channel.
```

Comment:
May be better to rephrase this, merging it with the other requirement for this call and avoiding generic pronouns. e.g.

```ruby
# Forcibly moves a user into a different voice channel.
# Only works if the bot has the permission needed and if the user is already connected to some voice channel on this server.
```